docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+```xml
+<allocation-stats totalBytes="524200" >
+  <allocated-bytes non-tlh="0" tlh="524200" arrayletleaf="0"/>
+</allocation-stats>
+```
+
+The following operations, each recorded by a `<gc-op>` element, run as part of the global cycle's increment:
+
+- `global mark`
+- `class unload`
+- `sweep`
+- `compact`
+
+WHAT INFO TO PULL OUT HERE, IF ANYTHING?
+
+```xml
+<gc-op id="2016" type="global mark" timems="357.859" contextid="2003" timestamp="2021-03-05T12:16:43.468">
+  <trace-info objectcount="37461962" scancount="37447916" scanbytes="828311396" />
+  <cardclean-info objects="311" bytes="22632" />
+  <finalization candidates="195" enqueued="2" />
+  <ownableSynchronizers candidates="2089" cleared="0" />
+  <references type="soft" candidates="3059" cleared="0" enqueued="0" dynamicThreshold="0" maxThreshold="32" />
+  <references type="weak" candidates="10797" cleared="0" enqueued="0" />
+  <references type="phantom" candidates="6" cleared="0" enqueued="0" />
+  <stringconstants candidates="10031" cleared="0"  />
+</gc-op>
+<gc-op id="2017" type="classunload" timems="0.123" contextid="2003" timestamp="2021-03-05T12:16:43.468">
+  <classunload-info classloadercandidates="25" classloadersunloaded="0" classesunloaded="0" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.123" scanms="0.000" postms="0.000" />
+</gc-op>
+<gc-op id="2018" type="sweep" timems="5.120" contextid="2003" timestamp="2021-03-05T12:16:43.474" />
+<gc-op id="2019" type="compact" timems="762.323" contextid="2003" timestamp="2021-03-05T12:16:44.236">
+  <compact-info movecount="8024461" movebytes="163375400" />
+  <remembered-set-cleared processed="777104" cleared="777104" durationms="2.188" />
+</gc-op>
+```
+
+The global cycle's increment ends. The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory allocation on the heap, similar to `<gc-start>`.
+
+
+```xml
+<gc-end id="2020" type="global garbage collect" contextid="2003" durationms="1126.788" usertimems="7971.788" systemtimems="1.000" stalltimems="1016.256" timestamp="2021-03-05T12:16:44.237" activeThreads="8">
+  <mem-info id="2021" free="1572864" total="838860800" percent="0">
+    <mem type="eden" free="1572864" total="1572864" percent="100" />
+    <pending-finalizers system="2" default="0" reference="0" classloader="0" />
+    <remembered-set count="874496" freebytes="29884416" totalbytes="33382400" percent="89" regionsoverflowed="0" regionsstable="0" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the snapshot at the beginning and end of this STW `global mark phase` subincrement shows that memory has been reclaimed and regions reassigned to create an empty eden space, equal to 1.5MB(1,572,864 B). In some cases, a global cycle also reclaims a small amount of memory from non-eden regions.
```

Comment:
Not sure that you meant by this and if this is really necessary 'In some cases, a global cycle also reclaims a small amount of memory from non-eden regions.'

Anyhow, you should state that amount of free memory reclaimed is not surprisingly low - we are here (global GC) because of tight memory conditions to start  with (and unless application makes some sudden turn in behavior there won't be much more free memory reclaimed).

Reply:
Replaced with:

_Comparing the snapshot at the beginning and end of this STW `global mark phase` subincrement shows that memory has been reclaimed and regions reassigned to create an empty eden space, equal to 1.5MB(1,572,864 B). Global cycles are triggered when memory conditions are tight, so it is not suprising that the global cycle is only able to reclaim a small amount of memory._